Implementation homework of Lesson 2 is finished

shmatko1095 opened the pull request:
Bash-script for add .c files to archive implemented 
@a-vodka please review

shmatko1095 committed:
Lesson01-Bash: Create shell script file

Empty shell script file is created

Signed-off-by: Oleksandr Shmatko <shmatko1095@gmail.com>

shmatko1095 committed:
Lesson01-Bash: Add command for /tmp/guesanumber dir creating

Signed-off-by: Oleksandr Shmatko <shmatko1095@gmail.com>

shmatko1095 committed:
Lesson01-Bash: Shell script according to second part of HW implementation

Add commnd for /realise branch create
Add command for copy .c files to /tmp/guesanumber
Add functions for archivating .c files from /tmp/guesanumber to release/guesanumber.tar.gz

Signed-off-by: Oleksandr Shmatko <shmatko1095@gmail.com>

a-vodka commented:
- Please, define temp dir as variable, and check if it exist
- Clear temp dir after archiving

shmatko1095 committed:
Directory defined as variables, add cleanup tmp dir after script finish

"tmp/guesanumber" defined as $tmpDir variable;
"realise" defined as $realiseDir variable;
add chiking of existing $tmpDir variable;
add remooving $tmpDir at scritp finish

Signed-off-by: Oleksandr Shmatko <shmatko1095@gmail.com>

a-vodka commented on 01-Git_Bash_Make/script.sh:
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+tmpDir=$tmpDir"tmp/guesanumber"
+realiseDir=$realiseDir"realise"
```
tmpDir and realiseDir have to be upper-case

a-vodka commented on 01-Git_Bash_Make/script.sh:
```diff
@@ -0,0 +1,16 @@
+   rmdir -p $tmpDir
+else
+   echo "dir can not be created"
+fi
```
Please, add empty line at end of the file

shmatko1095 committed:
Add empty line to the end of file

Empty line is added to the end of script

Signed-off-by: Oleksandr Shmatko <shmatko1095@gmail.com>

a-vodka commented:
You have fixed only second node. What about first one?

shmatko1095 committed:
tmpDir and realiseDir changed to be upper-case

Signed-off-by: Oleksandr Shmatko <shmatko1095@gmail.com>

shmatko1095 commented:
Fixed, @a-vodka , please review
